### PR TITLE
Add devise PR

### PR DESCRIPTION
**Pull request by kkasaija**
### Milestone reached

- Can register a new user.
- User logs in with a combination of email and password.
- Hashed password stored in the database.
- Ask for confirmation of email.
- Can reset password.

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <div class="field">
+    <%= f.label :Password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
```
Comment (Whoistolu) [inline app/views/devise/registrations/new.html.erb]: - [ ] [REQUIRED] Great job so far @Kasaija-Kenneth , however, you need to ensure that the views in devise for the registration and login have been modified to match the wireframes and their styling. Kindly resolve this before asking for another review.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2023-02-10 at 19 08 53 | Screenshot 2023-02-10 at 18 58 48

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Highlight
✔️ Removed `current_user` method in `ApplicationController`
✔️ Devise has been setup

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Kasaija-Kenneth, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Apply the change requested by the previous review👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Thanks @OLIPLICHE

**Pull request merged**